corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/utils.js

```diff
@@ -462,7 +462,7 @@ hqDefine("cloudcare/js/formplayer/utils/utils", function () {
     Utils.startIntervalSync = function (appId, attemptedRestore) {
         hqRequire(["cloudcare/js/formplayer/app"], function (FormplayerFrontend) {
             let currentApp = FormplayerFrontend.getChannel().request("appselect:getApp", appId),
-                customProperties = currentApp.attributes.profile.custom_properties;
+                customProperties = currentApp?.attributes?.profile.custom_properties || {};
```

Comment:
Assuming the implication here is that if `attributes` exists, `profile` always does (so it's safe to access `custom_properties`)?

Reply:
good point, I'm pretty sure that's true but made it more defensive just in case [40d700c](https://github.com/dimagi/commcare-hq/pull/34161/commits/40d700c1802116dc4750a699ae115122f26d63da)